Sources/Network/URLSession+Distribute.swift

```diff
@@ -22,6 +23,9 @@ extension URLSession {
           from: data
         ).message
       ) ?? "Unknown error"
+      if errorMessage == "Login required" {
```

Comment:
Can we use the status code like 401/403 instead? I'm not sure if API gateway lets you customize the text message here